system/services/admin/DataManagerService.cfc

```diff
@@ -813,8 +816,25 @@ component {
 						}
 
 						if ( _propertyIsSearchable( field, objName ) ) {
-							filter &= delim & fullFieldName & " like :#paramName#";
-							delim = " or ";
+							var fieldEnumName = poService.getObjectPropertyAttribute(
+								  objectName    = objName
+								, propertyName  = field
+								, attributeName = "enum"
+							);
+
+							if ( !isEmpty( fieldEnumName ) ) {
```

Comment:
This enum match should be done _in addition to_ the normal LIKE comparison if it's an enum, not instead of.